solutions/control flow/perimeter-of-a-rectangle.js

```diff
@@ -0,0 +1,21 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+    idx++;
+    return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+length = parseInt(readLine());
+breadth = parseInt(readLine());
```

Comment:
@prateek-mehra Please use `let` to declare the variables. Also, please change the folder name as pointed out by Pavan.